packages/angular_devkit/build_angular/src/builders/dev-server/vite-server.ts

```diff
@@ -12,28 +12,40 @@ import type { json, logging } from '@angular-devkit/core';
 import type { Plugin } from 'esbuild';
```

Comment:
We could also go with another approach, where we inject the `define` as globals in the polyfills file, through the banner option, however there might be cases were there are no polyfils.